specification/appendix/commitment_discounts.md

```diff
@@ -0,0 +1,274 @@
+# Commitment Discounts
+
+A commitment discount is a billing discount that offers reduced rates on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term. Commitment discounts typically consist of a set of purchase and usage records within cost and usage datasets.
+
+## Purchasing
+
+While customers are bound to the term of a commitment discount, cloud-service providers, or CSPs, offer various payment options before and/or during the term:
+
+* *Upfront* - The commitment discount is paid in full before the term begins
+* *Monthly* - The commitment discount is paid monthly over the term.
+* *Hybrid* (1/2 Upfront & 1/2 Monthly) - Half of the commitment discount is paid in advance and half is paid monthly over the term.
+
+For example, if a customer buys a spend-based commitment discount for 1 year, with a $1.00 hourly commitment, and pays with the hybrid option, the commitment discount's payment consists of both:
```

Comment:
This is the first mention of the term `spend-based`.  Given that this page is the explanation of a commitment discount, it should be written with the assumption that the reader brings no previous understanding with them.  This term deserves a definition earlier if you are going to use it here.

Reply:
I think by that logic, we would also need add definitions for:
- billing period
- charge period
- amortization
- term
...

Other than redefining `commitment discount` in the first sentence, I would rather link the glossary to the appendix rather than regurgitate definitions in 2 places. 

Thoughts?